css/main.css

```diff
@@ -0,0 +1,35 @@
+.cell {
+    width: 80px;
+    height: 80px;
+    border: lightpink solid 1px;
+    font-size: 40px;
+}
+.center {
+    text-align: center;
+    margin: 0 auto;
+}
+.primary-color {
+    color: lightpink;
+}
+.red {
+    color: red;
+}
+.green {
+    color: green;
+}
+.btn {
+    background-color: lightpink !important;
```

Comment:
Using !important is not good practice. https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception

Reply:
checked the rules from the link. as !important was used to override standard colour of bootstrap with light-pink and you said it's ok, no changes added to the code.